ui/store/institutional/institution-background.ts

```diff
@@ -142,7 +142,7 @@ export function mmiActionsFactory() {
     return () => {
       callBackgroundMethod(name, [payload], (err) => {
         if (isErrorWithMessage(err)) {
-          throw new Error(err.message);
+          throw new Error(err.data?.cause?.message || err.message);
```

Comment:
Open for suggestions for if/where the `err.data?.cause?.message || err.message` should be broken as a `getErrorMessage` library function.

Reply:
seems like a good idea

Reply:
Hmm, what is the purpose of wrapping the error in a new Error? As opposed to not catching it, or re-throwing.